configs/platforms/cumulus-2.2-amd64.rb

```diff
@@ -18,9 +28,8 @@
 echo 'deb http://osmirror.delivery.puppetlabs.net/debian/ wheezy main
 deb http://osmirror.delivery.puppetlabs.net/debian/ wheezy-updates main' >> /etc/apt/sources.list
 apt-get update -qq
-apt-get install -qy --no-install-recommends build-essential make quilt pkg-config debhelper devscripts rsync
+apt-get install -qy --no-install-recommends build-essential make quilt pkg-config debhelper devscripts rsync #{packages.join(' ')}
```

Comment:
Nitpick, but would be nice to include `build-essential`, `make`, etc. in the packages array too.